CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ if(DLAF_WITH_CUDA AND DLAF_WITH_HIP)
 endif()
 if(DLAF_WITH_CUDA)
   if(NOT CMAKE_CUDA_ARCHITECTURES)
-    set(CMAKE_CUDA_ARCHITECTURES "70;80;90" CACHE STRING "Cuda architectures" FORCE)
+    message(WARNING "Using default CMAKE_CUDA_ARCHITECTURES=${CMAKE_CUDA_ARCHITECTURES}.")
```

Comment:
Does ${CMAKE_CUDA_ARCHITECTURES} have a value here?

Reply:
This was my understanding from the documentation but I think I still had some cached variables in my test. This is only available after the `CUDA` language is enabled. I'll swap it.